Something/WebApplication/wwwroot/js/apps/authentication/components/Splash/Splash.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+import "./Splash.scss";
+
+export class Splash extends React.Component {
+    public render(): JSX.Element {
+        return (
+            <div className="Splash pane ant-col-md-12">
```

Comment:
it should be `pane ant-col-md-12 Splash` override classes should always be last

Reply:
Will do.